app/src/main/java/com/microsoft/device/samples/dualscreenexperience/presentation/catalog/item/CatalogPage1Fragment.kt

```diff
@@ -125,7 +125,7 @@ class CatalogPage1Fragment : Fragment() {
                 R.id.horizontal_fold, ConstraintSet.TOP, foldingMargin
             )
 
-            set.setVisibility(R.id.horizontal_fold, View.VISIBLE)
+            set.setVisibility(R.id.horizontal_fold, View.INVISIBLE)
```

Comment:
Is this change intentionally or by mistake?

Reply:
Intentionally, because if the folding feature is visible there will be a line shown on the screen. We don't need to show the line, just to constrain other views based on the folding feature's position, so INVISIBLE works better